SMLHelper/Patchers/CustomFishPatcher.cs

```diff
@@ -0,0 +1,68 @@
+﻿
+
+namespace SMLHelper.V2.FishFramework
```

Comment:
Could this be merged with `SMLHelper.V2.Patchers`? The handler is already in the appropriate namespace.

Reply:
I did merge it but forgot to change the namespace -_-